<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultsWidget</class>
 <widget class="QWidget" name="DefaultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifications</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbEnableMods">
     <property name="text">
      <string>Apply default image modifications</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbScale">
     <property name="title">
      <string>Scaling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbUpScale">
        <property name="text">
         <string>Scale image to screen size, if smaller, up to factor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KIntNumInput" name="sbMaxUpScaleFactor">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbDownScale">
        <property name="text">
         <string>Shrink image to screen size, if larger</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbGeometry">
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbFlipVertically">
        <property name="text">
         <string>Flip vertically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbFlipHorizontally">
        <property name="text">
         <string>Flip horizontally</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbRotate">
          <property name="text">
           <string>Rotate image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboRotate">
          <item>
           <property name="text">
            <string>0 Degrees</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90 Degrees</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>180 Degrees</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>270 Degrees</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="gbAdjust">
     <property name="title">
      <string>Adjustments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="KIntNumInput" name="sbBrightness">
        <property name="label">
         <string>Brightness:</string>
        </property>
        <property name="minimum">
         <number>-256</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntNumInput" name="sbContrast">
        <property name="label">
         <string>Contrast:</string>
        </property>
        <property name="minimum">
         <number>-256</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KIntNumInput" name="sbGamma">
        <property name="label">
         <string>Gamma:</string>
        </property>
        <property name="minimum">
         <number>-256</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="sliderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="gbPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0">
      <property name="margin">
       <number>10</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lbImOrig">
        <property name="text">
         <string>Original</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="lbImFiltered">
        <property name="text">
         <string>Modified</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>375</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
